tuf/src/database.rs

```diff
@@ -152,6 +192,45 @@ impl<D: DataInterchange> Database<D> {
             .unwrap_or(0)
     }
 
+    /// Verify and update metadata.
```

Comment:
Add a note about what the return type means, which in this case is that we've updated some metadata.

Reply:
Done